core/src/test/scala/com.mediative.sparrow/DataFrameReaderTest.scala

```diff
@@ -168,58 +153,12 @@ class DataFrameReaderTest extends FreeSpec with BeforeAndAfterAll {
     }
 
     def testFailure[T: RowConverter: ClassTag](json: Array[String], expected: NonEmptyList[String]) = {
+      val sqlContext = new SQLContext(sc)
       val df = sqlContext.jsonRDD(sc.parallelize(json))
 
       assert(toRDD[T](df) == expected.failure)
     }
 
-    // To get DataFrame#toRDD usage.
-    import com.mediative.sparrow.syntax.df._
-
-    "round-trip an object containing an Int and a String from RDD to DataFrame back to RDD" in {
```

Comment:
I suggest we move pending tests to a separate "spec" called `LimitationsSpec`, where each case should document why it is there and refer to any tickets that are planned to resolve them.